### PR TITLE
Conda Newsletter # 2

### PR DESCRIPTION
**Pull request by Pinak-Datta**
### Description
First draft of a conda newsletter Issue # 2. Includes latest releases, upcoming events, some information about latest blogs etc.

### CODE REVIEW

File: blog/2023-09-08-newsletter.md
Comment (travishathaway) [inline blog/2023-09-08-newsletter.md:5]:
```suggestion
    Upcoming Events, conda-store brief, Is conda free?, pre and post plugin hooks, Recent releases, Travis' latest talk on Youtube.
```

With the name "Travis", the possessive just ends with a single `'` and not `'s`; Trust me on this 😉.

File: blog/2023-09-08-newsletter.md
Comment (travishathaway) [inline blog/2023-09-08-newsletter.md:177]: I think it would be better to instead embed the YouTube video as an iframe. Did you experiment with that approach?

```
<iframe width="560" height="315" src="https://www.youtube.com/embed/d40tBcqopAI?si=NN62GyGLvbEgzqpC" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

File: blog/2023-09-08-newsletter.md
Comment (travishathaway) [inline blog/2023-09-08-newsletter.md:179]: I think this should read "...by conda maintainer Travis Hathaway".

First, in the conda community, there is actually a pretty famous Travis already (Travis Oliphant). He was pivotal in the creation of the conda tool, so I don't want readers to be confused.

Second, adding that information there gives more context to the reader who probably has no idea who I am.

File: blog/2023-09-08-newsletter.md
Comment (travishathaway) [inline blog/2023-09-08-newsletter.md:181]:
```suggestion
In this talk, Travis Hathaway explores the art of **designing plugin-friendly Python applications**. He emphasizes the importance of well-defined contracts between core software and plugins, fostering modular organization.
```

File: blog/2023-09-08-newsletter.md
Comment (travishathaway) [inline blog/2023-09-08-newsletter.md:183]:
```suggestion
Throughout the session, he demonstrates these principles using the pluggy library. He also shares a case study on conda, showcasing how it's adapting its codebase to embrace plugin-friendly practices.
```

File: blog/2023-09-08-newsletter.md
Comment (travishathaway) [inline blog/2023-09-08-newsletter.md:30]:
```suggestion
But that's not all! We'll also highlight one [job opportunity](#Job-still-Open-at-sfbi!) and share news about [Travis Hathaway's recent talk](  #Writing-Plugin-Friendly-Python-Applications:-by-Travis), now available for public consumption.
```

File: blog/2023-09-08-newsletter.md
Comment (travishathaway) [inline blog/2023-09-08-newsletter.md:46]:
```suggestion
Community member Dave Clements will be leading a tutorial on publishing open-source software to conda-forge. Conda-forge is a thriving community-managed channel within the conda ecosystem, boasting over 20,000 packages and serving billions of downloads annually. In this hands-on tutorial, you'll grasp the fundamentals of software packaging, understand the conda ecosystem, and explore the process of preparing and publishing your software on conda-forge.
```

File: blog/2023-09-08-newsletter.md
Comment (travishathaway) [inline blog/2023-09-08-newsletter.md:72]:
```suggestion
For more information, you can also check out [**Dave Clements' post on PackagingCon**](https://conda.org/blog/2023-07-25-packagingcon/). 
```

File: blog/2023-09-08-newsletter.md
Comment (travishathaway) [inline blog/2023-09-08-newsletter.md:104]:
```suggestion
[**In Dave Clements' latest blog post**](https://www.anaconda.com/blog/is-conda-free#summarize), he clarifies the question that's on everyone's mind: "*Is conda truly free?*" Here's a brief overview:
```

File: blog/2023-09-08-newsletter.md
Comment (travishathaway) [inline blog/2023-09-08-newsletter.md:115]:
```suggestion
For an in-depth look at conda's free offerings and the finer details, dive into [**Dave Clements' full blog post here**](https://www.anaconda.com/blog/is-conda-free#summarize).
```

File: blog/2023-09-08-newsletter.md
Comment (travishathaway) [inline blog/2023-09-08-newsletter.md:121]:
```suggestion
[**Key Highlights from Travis Hathaway's Blog:**](https://conda.org/blog/2023-07-31-latest-conda-release-includes-new-plugin-hooks/)
```

File: blog/2023-09-08-newsletter.md
Comment (travishathaway) [inline blog/2023-09-08-newsletter.md:127]:
```suggestion
For detailed implementation examples and insights into these new plugin hooks, [dive into Travis Hathaway's blog post here](https://conda.org/blog/2023-07-31-latest-conda-release-includes-new-plugin-hooks/).
```

File: blog/2023-09-08-newsletter.md
Comment (kathatherine) [inline blog/2023-09-08-newsletter.md:65]:
```suggestion
 - **Where**: Berlin (Hybrid, so online is an option too!)
```

File: blog/2023-09-08-newsletter.md
Comment (kathatherine) [inline blog/2023-09-08-newsletter.md:112]:
```suggestion
    -    Organizations with fewer than 200 people can use them for free.
```

File: blog/2023-09-08-newsletter.md
Comment (kathatherine) [inline blog/2023-09-08-newsletter.md:145]:
```suggestion
[**constructor 3.4.5**](https://github.com/conda/constructor/releases/tag/3.4.5): Constructor 3.4.5 enhances support for .yaml and .txt files in the environment_file key. This release also ensures accurate conda metadata when creating installers from existing environments on Windows.
```
Breaking that long sentence up. I had a hard time parsing it.

File: blog/2023-09-08-newsletter.md
Comment (kathatherine) [inline blog/2023-09-08-newsletter.md:147]:
```suggestion
[**conda-libmamba-solver 23.7.0**](https://github.com/conda/conda-libmamba-solver/releases/tag/23.7.0): Libmamba solver 23.7.0 introduces local channel integrations in conda-build, fixes for namespaced settings, CLI argument handling, authentication in package downloads, and improved handling of channels in conda-lock, among other enhancements and bug fixes.
```

File: blog/2023-09-08-newsletter.md
Comment (kathatherine) [inline blog/2023-09-08-newsletter.md:171]:
```suggestion
Join in revolutionizing genomics research! SFBI is looking for a Software Development Engineer to enhance its flagship tool, REPET. 
```

File: blog/2023-09-08-newsletter.md
Comment (kathatherine) [inline blog/2023-09-08-newsletter.md:159]:
```suggestion
## [Job still Open at SFBI!](https://www.sfbi.fr/emplois/offre/202308020700-cdd-ingenieur-e-en-developpement-logiciel)
```

File: blog/2023-09-08-newsletter.md
Comment (kathatherine) [inline blog/2023-09-08-newsletter.md:173]:
```suggestion
**Your mission**: Improve portability and scalability. If you have skills in Python, conda, Snakemake, and more, apply by September 20, 2023. [**Learn more**](https://www.sfbi.fr/emplois/offre/202308020700-cdd-ingenieur-e-en-developpement-logiciel).
```

**Review by kathatherine — CHANGES_REQUESTED** [review]
Thank you so much for compiling all this together. It's looking good!

File: blog/2023-09-08-newsletter.md
Comment (Pinak-Datta) [reply blog/2023-09-08-newsletter.md:179]: Okay!
Let me know if "Writing Plugin Friendly Python Applications: by Travis ( A Conda Maintainer)", works for you!

File: blog/2023-09-08-newsletter.md
Comment (Pinak-Datta) [reply blog/2023-09-08-newsletter.md]: Done!

File: blog/2023-09-08-newsletter.md
Comment (Pinak-Datta) [reply blog/2023-09-08-newsletter.md:177]: For some reason, the iframe tag isn't working. See if the workaround works for you!

File: blog/2023-09-08-newsletter.md
Comment (Pinak-Datta) [reply blog/2023-09-08-newsletter.md]: Done!

File: blog/2023-09-08-newsletter.md
Comment (Pinak-Datta) [reply blog/2023-09-08-newsletter.md]: Done

File: blog/2023-09-08-newsletter.md
Comment (Pinak-Datta) [reply blog/2023-09-08-newsletter.md]: Changed that !

File: blog/2023-09-08-newsletter.md
Comment (Pinak-Datta) [reply blog/2023-09-08-newsletter.md]: True, Changed now!

File: blog/2023-09-08-newsletter.md
Comment (Pinak-Datta) [reply blog/2023-09-08-newsletter.md]: Done!

File: blog/2023-09-08-newsletter.md
Comment (Pinak-Datta) [reply blog/2023-09-08-newsletter.md]: Done!

File: blog/2023-09-08-newsletter.md
Comment (Pinak-Datta) [reply blog/2023-09-08-newsletter.md]: Done!

File: blog/2023-09-08-newsletter.md
Comment (Pinak-Datta) [reply blog/2023-09-08-newsletter.md]: Done!

File: blog/2023-09-08-newsletter.md
Comment (Pinak-Datta) [reply blog/2023-09-08-newsletter.md]: Done!

File: blog/2023-09-08-newsletter.md
Comment (Pinak-Datta) [reply blog/2023-09-08-newsletter.md]: Done!

File: blog/2023-09-08-newsletter.md
Comment (Pinak-Datta) [reply blog/2023-09-08-newsletter.md]: Done!

File: blog/2023-09-08-newsletter.md
Comment (Pinak-Datta) [reply blog/2023-09-08-newsletter.md]: Removed the unnecessary whitespace.

File: blog/2023-09-08-newsletter.md
Comment (travishathaway) [reply blog/2023-09-08-newsletter.md]: "Writing Plugin Friendly Python Applications: by Travis Hathaway ( A Conda Maintainer)"

You need to use full names.

File: blog/2023-09-08-newsletter.md
Comment (travishathaway) [reply blog/2023-09-08-newsletter.md:177]: Try renaming the file to have the suffix ".mdx". I have already embedded an iframe elsewhere on the website: https://github.com/conda-incubator/conda-dot-org/blob/main/community/calendar.mdx

**Comment by travishathaway** [comment]
@Pinak-Datta,

Can you please add Bianca Henderson's talk to the newsletter? She also gave a talk recently at PyCon US that we would like highlight (https://youtu.be/OeSv0_HesaU?si=8aCREXFO1Jg6c9Cj)

File: blog/2023-09-08-newsletter.md
Comment (Pinak-Datta) [reply blog/2023-09-08-newsletter.md]: yup, that worked!
Thanks !

File: blog/2023-09-08-newsletter.md
Comment (Pinak-Datta) [reply blog/2023-09-08-newsletter.md]: Done!

**Comment by Pinak-Datta** [comment]
> @Pinak-Datta,
> 
> Can you please add Bianca Henderson's talk to the newsletter? She also gave a talk recently at PyCon US that we would like to highlight (https://youtu.be/OeSv0_HesaU?si=8aCREXFO1Jg6c9Cj)

Added a small brief, and the embedded video!

**Review by kathatherine — APPROVED** [review]
Looking good to me! If @tnabtaf signs off, I think this is ready to go.

File: blog/2023-09-08-newsletter.mdx
Comment (kathatherine) [inline blog/2023-09-08-newsletter.mdx:3]: @Pinak-Datta, before we merge, might be good to change the dates everywhere to be closer to the actual date of the newsletter release. 2023-09-19 or 2023-09-20 would be fine.

Comment (Pinak-Datta) [reply blog/2023-09-08-newsletter.mdx]: Changed that.

**Pull request merged**
